src/app/components/plugin-manager-settings.js

```diff
@@ -126,8 +130,8 @@ export class PluginManagerSettings {
 
   render () {
     return yo`
-    <footer class="navbar navbar-light bg-light ${css.permissions}">
-      <button onclick="${() => this.openDialog()}" class="btn btn-info">Settings</button>
+    <footer class="bg-light ${css.permissions} remix-bg-opacity">
```

Comment:
I think this definition of remix-bg-opacity should be defined in the csjs of this file and not in the bootstrap theme.  Let me know if you think there is an advantage to leaving it in the theme's css.